openapps/opencoap/opencoap.h

```diff
@@ -245,7 +245,7 @@ uint8_t opencoap_options_parse(uint8_t* buffer,
     uint8_t bufferLen,
     coap_option_iht* options,
     uint8_t* optionsLen);
-coap_option_iht* opencoap_find_option(coap_option_iht* array, uint8_t arrayLen, coap_option_t option); 
+uint8_t opencoap_find_option(coap_option_iht* array, uint8_t arrayLen, coap_option_t option, uint8_t* startIndex); 

```

Comment:
Why do you use `startIndex`? Would it not be possible to loop through the options and count the number of occurrences with only the first three parameters?